pki/certificates_test.go

```diff
@@ -41,8 +39,6 @@ func TestMain(m *testing.M) {
 	os.Exit(result)
 }
 
-var containerIDs []string
```

Comment:
Do we still need the `TestMain` above this, and the `setup` and `teardown` below this? Seems like they don't do anything now.